people.qmd

```diff
@@ -0,0 +1,724 @@
+# People
+
+## Team
+
+## Contributors
+
+
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+
+All contributions to this project are gratefully acknowledged using the [`allcontributors` package](https://github.com/ropenscilabs/allcontributors) following the [all-contributors](https://allcontributors.org) specification. Contributions of any kind are welcome!
+
+<table>
```

Comment:
Could we render it on the fly when rendering the whole website rather than having the re-generate the HTML code here each time?

I often use `knitr::knit_expand()` on child documents to create this kind of listing from a data frame. See, e.g., https://github.com/epiverse-trace/epiverse-trace.github.io/blob/main/_templates/hexwall_element.Rmd for the hexwall.

Reply:
Theoretically yes!

Practically, the way I was running [`allcontributors::add_contributors()`]() is a bit limiting and bloated for running it so frequently in the action. Specifically: `allcontributors` requires local copies of the repo's inspected to extract the contributors. 

I looked into it a bit more and found out that we can collect the data with less bloat using [`allcontributors::get_contributors_repo()`](https://docs.ropensci.org/allcontributors/reference/get_contributors.html). 

I will look into this and adjust - thanks!